<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListWidget" name="p2pListView"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="lineEditAddress"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="text">
         <string>Hello</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendBtn">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditLocalAddr"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>403</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="actionSelfTest"/>
    <addaction name="actionDefault_File"/>
    <addaction name="actionRequire_All"/>
    <addaction name="actionRequire_NAT"/>
    <addaction name="actionOnnRequire"/>
   </widget>
   <addaction name="menuTest"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSelfTest">
   <property name="text">
    <string>SelfTest</string>
   </property>
  </action>
  <action name="actionDefault_File">
   <property name="text">
    <string>Default config file</string>
   </property>
  </action>
  <action name="actionRequire_All">
   <property name="text">
    <string>Require All</string>
   </property>
  </action>
  <action name="actionRequire_NAT">
   <property name="text">
    <string>Require NAT</string>
   </property>
  </action>
  <action name="actionOnnRequire">
   <property name="text">
    <string>OnnRequire</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
